lib/open_food_network/i18n_inflections.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+module OpenFoodNetwork
```

Comment:
I'd love to kill this namespace. First by moving the OFN related services to app/services (most of them). We could create a generic lib folder like lib/utils for these generic classes and existing generic single classes like Locking, Paperclippable, and RackRequestBlocker. What do you think?

Reply:
Hm, yes, it would be nice to get rid of the namespace. But we need to come up with better namespaces. I don't find `Util` much better. We could use `I18n::Inflections` here but it could look like it's part of the official I18n library. It's just our add-on.

Moving files into their own namespaces would really help to see their independence. And we could create gems if we wanted to share those parts with other projects.

Reply:
I see what you mean. Util is not great but I think lib/util is better than lib/OpenFoodNetwork because it makes it obvious the code is not dependent on OFN code.
lib/utils would just be for those small util class (like Inflections).
I think a separate folder under lib would be for a group of related classes that are independent of OFN like we have in Stripe.
In Java the folder lib/common is used for these classes.

Probably a topic for discourse, not this PR :-)